docs/DeveloperGuide.adoc

```diff
@@ -572,6 +572,33 @@ Also creates coupling between the `Autocompleter` and `CommandBox`. +
 **Pros:** Easier to test since `CommandBoxTest` has already been set up and implemented. +
 **Cons:** `CommandBox` class now has multiple responsibilities, which violates SRP. +
 
+=== Maintain sorted
```

Comment:
Is there a better way you can name this? Like sorted mechanism (try to make it similar to the headings for the other features, which means ____ mechanism)